test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -35,7 +35,7 @@
  * @run main/othervm/timeout=600
  *     -Dsun.net.httpserver.nodelay=true
  *     -Dsun.net.client.defaultConnectTimeout=5000
- *     -Dsun.net.client.defaultReadTimeout=5000
+ *     -Dsun.net.client.defaultReadTimeout=50000
```

Comment:
For this one, it might be simpler to just drop the setting of the defaultReadTimeout.

Reply:
Done